plugins/auth-backend/src/providers/types.ts

```diff
@@ -124,6 +127,7 @@ export type AuthProviderFactoryOptions = {
   globalConfig: AuthProviderConfig;
   config: Config;
   logger: Logger;
+  tokenManager: TokenManager;
   tokenIssuer: TokenIssuer;
```

Comment:
The naming here seems a bit confusing. I can use a more explicit name like `serverTokenManager`, but open to suggestions. Perhaps we need some abstraction to encapsulate both of these objects?

Reply:
Just because I'm a bit confused, what two objects? :grin:

I feel the naming is fine though, the type is `TokenManager`, so makes sense?

Reply:
Referring to `tokenIssuer` vs `tokenManager`, considering both have a way to "issue tokens". Maybe not a big deal because we only expose `tokenIssuer` on the resolver context object